.github/workflows/android.yml

```diff
@@ -21,7 +21,4 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Run Tests
-        env:
```

Comment:
@thedevyansh FYI: you will not need to worry about dealing with these secrets in your PRs any longer!